src/constructs/ecs/ecs-task.ts

```diff
@@ -114,6 +114,13 @@ export interface GuEcsTaskProps extends AppIdentity {
   customTaskPolicies?: PolicyStatement[];
   environmentOverrides?: TaskEnvironmentVariable[];
   storage?: number;
+  /**
+   * Whether to give the task IAM role access to the account's dist bucket.
+   * This is enabled by default to avoid breaking changes, but consumers of this pattern
+   * that pull a container image that doesn't need extra dependencies from S3
+   * shoud set this value to `false`.
+   */
+  enableDistPolicy?: boolean;
```

Comment:
sorry could you make this enableDistributable policy - realise you've just copied my lazy variable name but might be a bit clearer